<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>CatPropDlgClass</class>
 <widget class="QDialog" name="CatPropDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CatPropDlg</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Category Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cpd_le_cat_name" />
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Default Directory:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="cpd_le_cat_directory" />
       </item>
       <item>
        <widget class="QToolButton" name="cpd_pb_open_directory" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item row="0" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Sub-Category:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLabel" name="cpd_lb_sub_category_num" >
         <property name="text" >
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>Downloaded File:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLabel" name="cpd_lb_total_file_num" >
         <property name="text" >
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Total Files:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="cpd_lb_downloaded_file_num" >
         <property name="text" >
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>Downloaded Size:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="cpd_lb_downloaded_size" >
         <property name="text" >
          <string>0 (null)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CatPropDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>287</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel" >
     <x>289</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>CatPropDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>310</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel" >
     <x>295</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
